runtime/oti/j9port.h

```diff
@@ -981,6 +983,7 @@ typedef struct J9CacheInfoQuery {
 #define J9PORT_ARCH_S390 OMRPORT_ARCH_S390
 #define J9PORT_ARCH_S390X OMRPORT_ARCH_S390X
 #define J9PORT_ARCH_RISCV OMRPORT_ARCH_RISCV
+#define J9PORT_ARCH_AARCH64 OMRPORT_ARCH_AARCH64
```

Comment:
Rather than add a new J9 macro, I suggest all `J9PORT_ARCH_*` be removed and `trclog.c` be updated to use the OMR names,